### ISSUE TITLE
Recruiting Garrisons

### ISSUE THREAD

**Issue by SomethingSimpl** [LordGolias/antistasi#27]
Recruiting garrisons detracts your money and HR but no troops are spawned.

Also if you keep clicking fast you can recruit way above your HR and money limits (still no troops spawn)

**Comment by SomethingSimpl** [comment]
After saving and loading the recruited garrisons appear. So the problem must be only after recruiting them during the session, they just simply wont spawn.

**Comment by SomethingSimpl** [comment]
Hm, it appears to be working every now and then, its not constant the way it doesnt spawn the units in. 

But the second part with fast clicking is still there.

**Comment by LordGolias** [comment]
Which garrison are you recruiting to? E.g. ATM if HQ is spawned and you recruit to HQ, the garrison will only spawn once the HQ is res-spawned again.

Regarding the fast clicking, yes, it is an error. :)

**Comment by SomethingSimpl** [comment]
Hm, we started capturing some ground a Hill emplacement and an AA outpost. Travelled back to HQ, and tried to recruit. The money and HR went down, numbers kept being added on the map but no soldiers were spawned there.

The next day I reloaded the mission and the troops spawned with appropriate numbers on the locations.

But after that I tried it again and this time all the troops spawned in right after I recruited them, so not really sure if it was just a one time thing.
